frontend/src/components/LegalNotice/LegalNotice.jsx

```diff
@@ -33,57 +35,45 @@ const LegalNotice = () => {
             <br />
             SIREN : 120 001 011
           </p>
-        </div>
-        <div>
-          <h3>Directeur de la publication</h3>
-          <p className="fr-mb-2w">
+        </Section>
+        <Section title="Directeur de la publication">
+          <span>
             MINISTÈRE DE L&lsquo;ENSEIGNEMENT SUPÉRIEUR,
             DE LA RECHERCHE ET DE L&lsquo;INNOVATION
-          </p>
-          <p className="fr-mb-2w">1 rue Descartes - 75231 Paris cedex 05</p>
-        </div>
-        <div>
-          <h3>Hébergement de la Plateforme</h3>
+          </span>
+          <p>1 rue Descartes - 75231 Paris cedex 05</p>
+        </Section>
+        <Section title="Hébergement de la Plateforme">
           Ce site est hébergé en propre par Scalingo SAS, 15 avenue du Rhin, 67100 Strasbourg, France.
-        </div>
-        <div id="accessibilite">
-          <h3>Accessibilité</h3>
-          <p className="fr-mb-2w">
+        </Section>
+        <Section title="Accessibilité">
+          <p>
             La conformité aux normes d’accessibilité numérique est un objectif ultérieur.
             En attendant, nous tâchons de rendre ce site
             accessible à toutes et à tous :
           </p>
-          <ul className="fr-list">
+          <ul className={styles.list}>
```

Comment:
Why not use "fr-list" style? (or at least keep the dots)

Before:
![before](https://user-images.githubusercontent.com/68006779/126962172-4846bb68-9c8e-41d0-baca-ccf61befe796.jpg)

After:
![after](https://user-images.githubusercontent.com/68006779/126962183-dc8cd030-41f0-4d8e-bb99-f10c9b128a3f.jpg)

Reply:
> Why not use "fr-list" style? (or at least keep the dots)
> 
> Before:
> ![before](https://user-images.githubusercontent.com/68006779/126962172-4846bb68-9c8e-41d0-baca-ccf61befe796.jpg)
> 
> After:
> ![after](https://user-images.githubusercontent.com/68006779/126962183-dc8cd030-41f0-4d8e-bb99-f10c9b128a3f.jpg)

I'm refering to this UX page : https://mure.invisionapp.com/console/share/E22KQWN1KF/630743966/play
and to disable the margin between each <li> the best way is styles.list .

Reply:
I keep the dots and used fr-list like before.